juan-aristizabal-bedoya/tarea-5-puzzle/game.js

```diff
@@ -0,0 +1,75 @@
+//takes all tiles with class .tile, an select the board
+const gameTiles = document.querySelectorAll(".tile");
+const gameBoard = document.querySelector("#game-board");
+
+//tiles array
+const gameState = [
+  [gameTiles[0], gameTiles[1], gameTiles[2], gameTiles[3]],
+  [gameTiles[4], gameTiles[5], gameTiles[6], gameTiles[7]],
+  [gameTiles[8], gameTiles[9], gameTiles[10], gameTiles[11]],
+  [gameTiles[12], gameTiles[13], gameTiles[14], gameTiles[15]],
+];
+//updates the board and tiles
+function render(gameBoard, gameState) {
+  gameState.forEach((row, rowIndex) => {
+    row.forEach((column, columnIndex) => {
+      column.style.top = `${rowIndex * 100}px`;
+      column.style.left = `${columnIndex * 100}px`;
+
+      gameBoard.appendChild(column);
+    });
+  });
+}
+
+function moveElement(element1, element2) {
+  const tempTop = element1.style.top;
+  const tempLeft = element1.style.left;
+
+  element1.style.top = element2.style.top;
+  element1.style.left = element2.style.left;
+
+  element2.style.top = tempTop;
+  element2.style.left = tempLeft;
+}
+
+render(gameBoard, gameState);
```

Comment:
@Juan-Aristizabal , make sure all functions are alphabetically sorted, and move this render call to the end of file